lib/public_key/test/pkits_SUITE.erl

```diff
@@ -331,8 +331,15 @@ not_before_invalid(Config) when is_list(Config) ->
 not_before_valid() ->
     [{doc,"Test valid periods"}].
 not_before_valid(Config) when is_list(Config) ->
-    run([{ "4.2.3", "Valid pre2000 UTC notBefore Date Test3 EE", ok},
-	 { "4.2.4", "Valid GeneralizedTime notBefore Date Test4 EE", ok}]).
+    {Year, Month, Day} = date(),
```

Comment:
This is not the best place to place the test, as this is the test suite implementing conformance tests according to
https://csrc.nist.gov/CSRC/media/Projects/PKI-Testing/documents/PKITS_v1_0_0.pdf

I would instead place the test in public_key_SUITE.  Maybe as part of the test case pkix_path_validation_root_expired or possibly a new one close to it. And use  public_key:pkix_test_data to generate certificate data instead of legacy erl_make_certs that is dependent on OpenSSL

Reply:
@IngelaAndin I have done the changes explained above :)

Reply:
I have also added the beginning of a `pubkey_cert_SUITE` that tests one of the exported functions.
Hopefully we can continue testing exported functions from that module from here on :)